### ISSUE TITLE
Regarding the result

### ISSUE THREAD

**Issue by maxzkn** [is-a-dev/register#15592]
Hi,

Looking at this [Pull Request](https://github.com/is-a-dev/register/pull/15582/commits/876efef450b9b59d12a34153c4d903152b03773f) (which was already merged), how my website should look now?

I can't access it via [mzykin.is-a.dev](mzykin.is-a.dev) (DNS_PROBE_FINISHED_NXDOMAIN), so I have a suspicion that I didn't provide my CNAME correctly? I can see that I can access [mzykin.com.is-a.dev](mzykin.com.is-a.dev) but the browser tells me that the Connection is not private and I can't access the website.

Did I write my CNAME incorrectly? Should I change it to just "mzykin" so that I could access my website via "mzykin.is-a.dev"?

Thank you

**Comment by hansanaD** [comment]
I had the same problem since now and i found this as the solution. give it a go and let know your result.

**Clear your browser cache:**

Browsers store temporary files and data, which can sometimes cause conflicts leading to automatic redirection. Clearing the cache helps eliminate such conflicts. To do this, follow these steps:

- **For Google Chrome:** Go to Settings > Privacy and Security > Clear Browsing Data > select "Cached images and files" > Clear Data.
- **For Mozilla Firefox:** Go to Options > Privacy & Security > Cookies and Site Data > Clear Data.
- **For Safari:** Go to Safari > Preferences > Privacy > Manage Website Data > Remove All.

Credits: https://www.quora.com/My-personal-website-is-being-redirected-to-another-website-automatically-How-can-I-solve-this-issue

**Comment by orxngc** [comment]
@hansanaD No, it doesn't work for me either so it can't be a cache issue.
@maxzkn On the is-a.dev side, you formatted and entered your records correctly, else I wouldn't have merged your PR. What you'll need to do is go into your DNS provider for mzykin.com and create a CNAME record that points to `mzykin.is-a.dev`. Then your problem should be fixed.

**Comment by maxzkn** [comment]
@hansanaD Thank you for your help, but this indeed isn't an issue in my particular case.
@orxngc Sorry to bother, but I'm having a hard time creating a correct record in my Cloudflare account. I can only create a record where `mzykin.com` points to `mzykin.is-a.dev`, but I assume this should the other way around, though I can't set it up. Maybe you had some experience with Cloudflare? Please see screenshots attached below!

**I can't register it like this:**

<img width="984" alt="dns1" src="https://github.com/user-attachments/assets/9e4584a9-08ff-48de-859d-6e3c1822538b">




**Only like this:**

<img width="984" alt="dns2" src="https://github.com/user-attachments/assets/43230876-20ec-40f9-8cef-2eda7aed7dba">




**And this is the result:**

<img width="1063" alt="dns3" src="https://github.com/user-attachments/assets/6fb1d09c-0c1f-49f7-830d-34c61fb96df0">

**Comment by orxngc** [comment]
@maxzkn  `name` should be set to `@`, and `target` should be set to `mzykin.is-a.dev`. If you want the is-a.dev domain to be set to a subdomain of mzykin.com, then replace the `@` in the name field with whatever you want the subdomain to be; for example, put `uwu` in the name field and `mzykin.is-a.dev` in the target field if you want `mzykin.is-a.dev` to redirect to `uwu.mzykin.com`.

**Comment by maxzkn** [comment]
@orxngc 

> name should be set to @, and target should be set to mzykin.is-a.dev.

So this is exactly what I did (please look at the second screenshot in my previous comment), but this just doesn't work:
![image](https://github.com/user-attachments/assets/7f4fe179-9108-444a-925d-81a33f9d73fe)

**Comment by orxngc** [comment]
This seems to be an SSL issue.. I forgot how this part works. @CuteDog5695 Do you know what's going on here?

**Comment by maxzkn** [comment]
I have a suspicion that it can be related to that my website is not working without `www` subdomain... seems like I have to somehow configure my apex domain first, so that website would be accessed by `mzykin.com` (without `www.`).

**Comment by CuteDog5695** [comment]
Did you even add SSL for the is-a.dev domain? Or are you just trying to connect the domain directly without SSL stuff?

**Comment by maxzkn** [comment]
@CuteDog5695 Thank you for your answer. So I try to follow your recommendation and register `is-a.dev` subdomain with my Cloudflare Page (along with already registered `mzykin.com`) and as you said I stumbled upon the DNS choice, but the only choice I have is to transfer DNS:

<img width="915" alt="Screenshot at Aug 19 14-24-26" src="https://github.com/user-attachments/assets/9b179d51-d8fb-4581-bf15-8dc07624a2e4">


> While doing it, if you stumble upon what dns you want to chose go with the non-cloudflare option

Once I click on `Begin DNS transfer`, I can see this view, but I don't really understand with which `non-cloudflare option` should I go and should I put `is-a.dev` or `mzykin.com` to an `Existing domain`?


<img width="604" alt="image" src="https://github.com/user-attachments/assets/a74f2355-ad05-4113-b013-4599590dd645">

**Comment by CuteDog5695** [comment]
Did you first go to your Cloudflare Page -> Settings -> Custom Domains?

**Comment by maxzkn** [comment]
Yes, I did:

<img width="813" alt="image" src="https://github.com/user-attachments/assets/f1337fc9-9179-43cb-9167-0d966810391a">


<img width="1127" alt="image" src="https://github.com/user-attachments/assets/8d1a24e4-22c5-476d-bfb9-2d0406aaea9e">

**Comment by CuteDog5695** [comment]
And you putted your is-a.dev domain? (mzykin.is-a.dev)

**Comment by maxzkn** [comment]
Oh sorry, I was adding just the `.is-a.dev`. Now when I added `mzykin.is-a.dev`, I clicked on `Configure DNS` as per your suggestion:

<img width="878" alt="image" src="https://github.com/user-attachments/assets/765eb1f3-78e0-4076-ab4e-81d7847c5949">

And now I see this view:

<img width="740" alt="image" src="https://github.com/user-attachments/assets/15a3bb4c-700f-4947-98ea-de5bcb7e34f0">


But my DNS provider is Cloudflare, as it is my domain registrar. So does it mean that I need to add a CNAME record in my DNS settings in Cloudflare with Name as `mzykin` and Target as `portfolio-website-d84.pages.dev` ?  I don't think so, because if I type `mzykin` in the Name field, then the root will be `mzykin.mzykin.com`. If I type `@`, then the Name will be the root itself (`mzykin.com`).

**Comment by CuteDog5695** [comment]
Didn't you want to add your is-a.dev domain? If you wanted for your is-a.dev domain to redirect to your main domain then you could of used redirect.pizza

**Comment by maxzkn** [comment]
No, my goal is to add `.is-a.dev` to `mzykin.com` (or `www.mzykin.com`, since I didn't configure my website to work without `www` subdomain yet) so that the URL would look like `mzykin.is-a.dev`

**Comment by CuteDog5695** [comment]
Sorry, but what are you trying to do with the is-a.dev domain? I'm a bit lost now

**Comment by maxzkn** [comment]
I bought a custom domain in Cloudflare, which is `mzykin.com`. I deployed my website to Cloudflare Pages and set `www` as a subdomain which points to my website's original URL:

<img width="1067" alt="image" src="https://github.com/user-attachments/assets/81a84b27-f7a0-40f1-96a2-186afffbb225">


I now have a website [www.mzykin.com](www.mzykin.com). I would like to add `.is-a.dev` as a top-level domain for my website, so that I could access it as `www.mzykin.is-a.dev`.

Ideally, that would be `mzykin.is-a.dev`, but as I said for that I need to set up my website to be accessed without `www` subdomain as well, but I can deal with this later.

**Comment by maxzkn** [comment]
**Update to everyone**: as the error `ERR_SSL_VERSION_OR_CIPHER_MISMATCH` indicates, it seems like I need to add a SSL certificate which would cover `.is-a.dev` subdomain, but it seems like I can't do it with a free Cloudflare plan, because it only protects `mzykin.com` and `*.mzykin.com`. So, if you want to cover additional domains with additional certificates, you need to upgrade to Advanced Certificate Manager which costs 10$/month, which I am not willing to do.

Unless I am mistaken and the problem lies somewhere else and not here, I think this thread can be closed now.
